gift-certificate-application/controller/src/main/java/com/epam/esm/controller/exception/ResponseBodyPreparator.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.controller.exception;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Map;
+
+public class ResponseBodyPreparator {
```

Comment:
Since this class doesn't prepare anything (it doesn't create any object in the "prepare" method) it's better to name like "ResponseBodyEnricher" or something like that, also please make it final and create a private constructor as it looks like an util class)